### PR TITLE
added new tokens

### PR DESCRIPTION
**Pull request by jan-magpie**
added vlpnp token

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/penpie exports TVL: 
        
 
 
    ethereum                  111.03 M
    arbitrum                  83.86 M
    staking                   10.96 M
    arbitrum-staking          8.87 M
    ethereum-staking          1.85 M
    bsc                       1.72 M
    bsc-staking               237.54 k
    optimism                  187.59 k
    optimism-staking          10.58 k
    
    total                    196.80 M

File: projects/penpie/index.js [hunk 1]
```diff
@@ -12,9 +12,12 @@ async function tvl(api) {
   });
   const poolInfos = await api.multiCall({ abi: 'function getPoolInfo(address) view returns ( uint256 emission, uint256 allocpoint, uint256 sizeOfPool, uint256 totalPoint)', calls: poolTokens, target: masterPenpie, })
   const symbols = await api.multiCall({ abi: 'erc20:symbol', calls: poolTokens, })
+  
   poolTokens.forEach((token, i) => {
-    if (symbols[i] === 'vlPenpie' || symbols[i] === 'mPendle' || symbols[i] === 'mPendleOFT' || symbols[i] === 'mPendleSV') {
-      token = PENDLE
+ 
+    if (symbols[i] === 'vlPenpie' ){
```
Comment (g1nt0ki) [inline projects/penpie/index.js:18]: isnt this already counted towards staking?

Comment (jan-magpie) [reply projects/penpie/index.js:18]: mpendle,pendle,penpie prices are now fetchable directly. hence removed that line and added an if condition only for vlpenpie which is locked version of penpie

Comment (g1nt0ki) [reply projects/penpie/index.js:18]: yeah, but we shouldnt count staked penpie as part of tvl, as it is already counted as part of staking?

**Comment by g1nt0ki** [comment]
fixed it here: https://github.com/DefiLlama/DefiLlama-Adapters/commit/8e892feb2fa482b3d4f369d061a7cb54f28af04c

**Pull request closed without merging**